homeassistant/components/netgear_lte.py

```diff
@@ -80,7 +83,18 @@ def get(self, config):
     host = lte_config[CONF_HOST]
     password = lte_config[CONF_PASSWORD]
 
-    eternalegypt = eternalegypt.LB2120(host, password)
-    lte_data = LTEData(eternalegypt)
+    websession = async_create_clientsession(
+        hass, cookie_jar=aiohttp.CookieJar(unsafe=True))
```

Comment:
You create for every device a new session. Can you not share the session over all devices?

Reply:
Good point. I moved the websession to the shared state and did some renames (in a separate commit) to reflect the change in responsibility.